tf_manager:
  ros__parameters:
    pose_topic: "/odom/ground_truth"
    input_gazebo_pose_topic: "/odom/gazebo"
    input_mujoco_pose_topic: "/odom/mujoco_odom"
    output_pose_topic: "/odom/ground_truth"
    mc_odom_topic: "/odom/mc_odom"
    uniform_odom_topic: "/odom/ground_truth"
    map_frame: "map"
    odom_frame: "odom"
    base_frame: "base_link"
    sleep_ms: 10  
    filter_nums: 20